src/drivers/serial/Rs232.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file Rs232.cpp
+ * @author Micah Switzer (mswitzer@cedarville.edu)
+ * @brief 
+ * @version 0.1
+ * @date 2019-09-30
+ * 
+ * @copyright Copyright (c) 2019
+ * 
+ */
+#include <drivers/serial/Rs232.hpp>
+
+Rs232::Rs232(uint16_t portNum, InterruptManager* interruptManager) :
+    InterruptHandler(interruptManager, portNum == COM1 ? COM1_IRQ : COM2_IRQ),
+    portBase(portNum) {
```

Comment:
Why is the portBase(portNum) still here?

Reply:
Because it's to keep track of which ports we're writing to